main.js

```diff
@@ -16,42 +17,70 @@ async function fetchFollowers(username) {
   }
```

Comment:
@raaj-oo7 This fetch function catch block should not throw an error.
Could you please explain why this function throws error?

Reply:
if network error or fetch operation error occurred that time function throws an error